zbus/src/connection/handshake/client.rs

```diff
@@ -179,7 +179,22 @@ impl Client {
 
         let can_pass_fd = self.common.socket_mut().read_mut().can_pass_unix_fd();
         if can_pass_fd {
-            commands.push(Command::NegotiateUnixFD);
+            // xdg-dbus-proxy can't handle pipelining, hence this special handling.
+            // FIXME: Remote this as soon as flatpak is fixed and fix is available in major distros.
```

Comment:
```suggestion
            // FIXME: Remove this as soon as flatpak is fixed and fix is available in major distros.
```

maybe also link to the xdg-dbus-proxy issue here for future reference